5.5/README.md

```diff
@@ -40,6 +40,8 @@ The following environment variables influence the MySQL configuration file. They
 |  `MYSQL_INNODB_LOG_BUFFER_SIZE` | The size of the buffer that InnoDB uses to write to the log files on disk | 8M (or 15% of available memory)
 |  `MYSQL_DEFAULTS_FILE`          | Point to an alternative configuration file                        |  /etc/my.cnf
 |  `MYSQL_BINLOG_FORMAT`          | Set sets the binlog format, supported values are `row` and `statement` | statement
+|  `MYSQL_LOG_QUERIES`            | Turns on logging of queries | 0
```

Comment:
"By default, the general query log is disabled. To specify the initial general query log state explicitly, use --general_log[={0|1}]. With no argument or an argument of 1, --general_log enables the log. With an argument of 0, this option disables the log. To specify a log file name, use --general_log_file=file_name."
from [DOC](http://dev.mysql.com/doc/refman/5.7/en/query-log.html)